tests/test_updown.py

```diff
@@ -4,6 +4,8 @@
 import tempfile
 from flask import url_for
 
+import ctadata
```

Comment:
it's not the same to import at the top, what if there is an exception during import, it will not be tested, it will fail during discovery

Reply:
Ok, I reverted this change